lib/helpers.js

```diff
@@ -60,8 +60,29 @@ exports.validateName = function (name) {
 };
 
 /**
+ * Check if relation has a different name from properties
+ * @param {Object} modelDefinition The model which has the relation
+ * @param {String} name The user input
+ * @returns {String|Boolean} The check result
```

Comment:
Since this function is async, you should describe the callback args, not the return value.

```js
/**
 * @param {Function(String|Boolean)} Callback to receive the check result.
 */
```